fcrepo-connector-file/src/test/java/org/fcrepo/connector/file/FedoraFileSystemConnectorTest.java

```diff
@@ -165,6 +166,28 @@ public void testGetDocumentByIdDatastream() {
     }
 
     @Test
+    public void testGetDocumentByIdObject() {
+        when(mockTranslator.getPrimaryTypeName(any(Document.class)))
+                .thenReturn(NT_FOLDER);
+        when(mockNameFactory.create(anyString())).thenReturn(
+                new BasicName("", tmpFile.getName()));
+
+        final Document doc = connector.getDocumentById("/" + tmpFile.getName());
+        assertNotNull(doc);
```

Comment:
Quite possibly. There I was simply mirroring the existing test cases you see above and below (`testGetDocumentByIdDatastream`, `testGetDocumentByIdContent`, etc)